merge.go

```diff
@@ -46,6 +52,16 @@ func (m *MergeConfig) ToString() (string, error) {
 		sb.WriteString(str)
 	}
 
+	if m.OnMatchWithMembers != nil {
+		str, err := m.OnMatchWithMembers.ToString()
+		if err != nil {
+			return "", err
+		}
+
+		sb.WriteString(" ON MATCH SET")
```

Comment:
As per the [official doc](https://neo4j.com/docs/cypher-manual/current/clauses/merge/#merge-merge-with-on-match-setting-multiple-properties), have one single `SET` should be sufficient.